### DISCUSSION TITLE
How can I clear my fastlane variables

### DISCUSSION THREAD

**Discussion by maxisme** [fastlane/fastlane#18609] [Q&A]
Hello,

How can I start afresh with my fastlane? I want to clear all my apple credentials so I can test locally as if I was in the CI.

Thank you

**Comment by Keshaunm** [comment]
Right-click on My Computer and select Properties.
Go to the Advanced system settings tab.
Click the Environment Variables button. The Environment Variables dialog opens.
Select the environment variable you want to delete and click Delete.
Repeat step 4 as many times as necessary.
Click OK.


[Kabosutoken](https://kabosucoin.net/)
